libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
This change is incorrect; if you remove this then the message that's sent won't be the error code from `rename()`.  I suggest replacing with:

```
t_save_errno = errno;
```